TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementCheckers.cs

```diff
@@ -7,19 +8,38 @@ namespace TypeCobol.Compiler.Sql.CodeElements
 {
     public static class SavepointStatementChecker
     {
-        public static void OnCodeElement(SavepointStatement savepointStatement,
-            CodeElementsParser.SavepointStatementContext context)
+        public static void OnCodeElement(SavepointStatement savepointStatement, CodeElementsParser.SavepointStatementContext context)
         {
             if (savepointStatement.Name != null)
             {
                 var savePointName = savepointStatement.Name.ToString();
                 if (savePointName.StartsWith("SYS", StringComparison.OrdinalIgnoreCase))
                 {
                     DiagnosticUtils.AddError(savepointStatement,
-                        "Invalid savepoint-name, it must not begin with 'SYS'.",
-                        context);
+                        "Invalid savepoint-name, it must not begin with 'SYS'.", context);
                 }
             }
         }
     }
+    public static class IntegrateAlterSequenceStatementChecker
```

Comment:
- naming `AlterSequenceStatementChecker`
- Checker classes should be internal, let's use this PR to change it for SavepointStatementChecker and AlterSequenceStatementChecker

Reply:
Ok for the visibility, now please change the name.